libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/MapboxCopilot.kt

```diff
@@ -45,8 +46,11 @@ import java.util.concurrent.CopyOnWriteArraySet
 @ExperimentalPreviewMapboxNavigationAPI
 object MapboxCopilot : MapboxNavigationObserver {
 
-    private var copilot: MapboxCopilotImpl? = null
+    @VisibleForTesting
+    internal var copilot: MapboxCopilotImpl? = null
     internal val pushStatusObservers = CopyOnWriteArraySet<PushStatusObserver>()
+    @VisibleForTesting
+    internal var isCopilotInitialized = false
```

Comment:
I think we don't need a flag. We can check `copilot` instead. If it is not null, then Copilot is initialized. Otherwise it isn't.